skymp5-server/cpp/server_guest_lib/MpActor.cpp

```diff
@@ -924,3 +924,28 @@ void MpActor::ReapplyMagicEffects()
     [](std::string_view fileName) { return fileName == "SweetPie.esp"; });
   ApplyMagicEffects(activeEffects, hasSweetpie, true);
 }
+
+std::pair<std::optional<uint32_t>, std::optional<uint32_t>>
```

Comment:
I don't like the return type. I understand that `std::pair` is required for `std::apply` which is.. well.. modern and fun

But I don't like that contributors of beautiful future skymp will have to learn function body in order to understand the return type.

Suggested alternatives:
- `std::vector<Inventory::Entry>`

In Skyrim, Worn may also mean two-handed weapon. Or it's Worn+WornLeft. I don't really remember.